src/c++/devices/corruptor/dmCorruptor.c

```diff
@@ -773,8 +773,8 @@ static struct target_type corruptorTargetType = {
 /**********************************************************************/
 int __init corruptorInit(void)
 {
-  STATIC_ASSERT(offsetof(CorruptorDevice, dev)
-                == offsetof(CommonDevice, dev));
+  BUILD_BUG_ON(offsetof(CorruptorDevice, dev)
```

Comment:
This is not specifically related to this change, but I think these BUILD_BUG_ON lines do not need to wrap. I think if they are one line instead of two, they will not be too long.
(You can change this or not change this, either way is fine.)